bodhi/server/scripts/manage_releases.py

```diff
@@ -27,17 +27,29 @@
     --pending-testing-tag f23-updates-testing-pending --override-tag f23-override --state pending
 """
 
-import sys
-import munch
-
 import click
 
 from bodhi.client import bindings
+from bodhi.client import (
+    cli,
+    add_options,
+    release_options,
+    create_release,
+    edit_release,
+    info_release
+)
 
 
 @click.group()
-def main():
-    """Create and manage releases in Bodhi."""
+@click.pass_context
+def main(ctx):
+    """
+    Create and manage releases in Bodhi.
+
+    (This utility has been deprecated. Please use 'bodhi releases' instead.)
+    """
+    click.echo("This utility has been deprecated. Please use 'bodhi releases' instead.")
+    ctx.forward(cli)
```

Comment:
It looks like the tests fail due to missing coverage on these two lines.

Reply:
Is there a way to test the main() function here? I believe that invoking it using CliRunner() will display its '--help' text and will not execute the code after it. (Reason maybe because it is a @click.group and not @click.command)